src/main/java/edu/kit/hci/soli/domain/Priority.java

```diff
@@ -0,0 +1,9 @@
+package edu.kit.hci.soli.domain;
+
+public enum Priority {
+    ONE,
```

Comment:
Should we assign concrete numbers to this enum so we can compare them using > (Or is this not java-esque enough?)
Also we need to clearify which priotity is the highest, is it really the highest number?

Reply:
Yes i guess we could change it to "highest", "medium", "lowest" and display in the ui something different